src/lib/daoBase.js

```diff
@@ -42,14 +45,21 @@ module.exports = class Base {
   }
```

Comment:
always return true?

Reply:
it returns `return this.ref.child(id).update(payload)` 

a promise.

Reply:
it returns `return this.ref.child(id).update(payload)` 

a promise.